### PR TITLE
CVE-2016-3092

### PR DESCRIPTION
**Pull request by HunterL**
Added information on CVE-2016-3092 which represented a vulnerability in the Apache Commons FileUpload dependency. The vulnerability allowed an attacker to set a long boundary string which would cause the FileUpload to use a significant amount more of CPU resources during parsing.

### CODE REVIEW

File: cves/CVE-2016-3092.yml [hunk 1]
```diff
@@ -208,4 +218,6 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    Input validation was improperly handled originally. Prior to the fix
+    boundary existence was checked, however the boundary being too long was not
```
Comment (gollb310) [inline cves/CVE-2016-3092.yml:223]: I would like some more expansion on the mistake answer. It feels a bit short and incomplete.

File: cves/CVE-2016-3092.yml [hunk 2]
```diff
@@ -122,7 +128,7 @@ discovered:
     If there is no evidence as to how this vulnerability was found, then you
     may leave the entries blank except for "answer", BUT please write down
     where you looked in "answer".
-  answer:
+  answer: 
   date:
   automated:
   contest:
```
Comment (gollb310) [inline cves/CVE-2016-3092.yml:134]: Do you have any information on its discovery? I believe some of these fields need to be filled in to pass the build as well.

**Review by gollb310 — COMMENTED** [review]
Some sections need expansion but overall everything else looks good. +3 upvotes.

**Comment by mesternefeld** [comment]
There are 7 files touched but there should only be one. It would be a good idea to keep the one you worked on

**Pull request closed without merging**